src/encoded/schemas/file.json

```diff
@@ -242,66 +242,105 @@
                 }
             ]
         },
+        "assembly": {
+            "not": {
+                "anyOf": [
+                    {
+                        "properties": {
+                            "output_type": {
+                                "enum": ["reads"]
+                            }
+                        }
+                    },
+                    {
+                        "properties": {
+                            "output_type": {
+                            "enum": ["rejected reads"]
+                            }
+                        }
+                    }
+                ]                        
```

Comment:
This line seems to have a bunch of empty space at the end.  Please remove.

Reply:
done